### PR TITLE
screenpipe plugin 001

### PR DESCRIPTION
**Pull request by m13v**

### CODE REVIEW

**Review by josancamon19 — CHANGES_REQUESTED** [review]
Pls include the json data in community-plugins.json.

Also the json is possibly wrongly formatted.

File: plugins/instructions/screenpipe/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Friend Integration Guide
+
+Get started with screenpipe:
```
Comment (josancamon19) [inline plugins/instructions/screenpipe/README.md]: Can you also include here, what's screenpipe and what this plugin does?

**Review by josancamon19 — CHANGES_REQUESTED** [review]

**Comment by m13v** [comment]
made updates/changes

**Comment by m13v** [comment]
resolved

**Review by josancamon19 — APPROVED** [review]

**Comment by m13v** [comment]
🥳

**Comment by ellaaicare** [comment]
## Correction Pipeline Analysis — April 22, 2026

### What I tested

Sent test payloads directly to the live n8n `conversation-correction` webhook endpoint. Results:

1. **Webhook IS reachable** — `POST https://n8n.ella-ai-care.com/webhook/conversation-correction` returns HTTP 202 ✅
2. **Immediate response**: `{\"status\":\"success\",\"trace_id\":\"...\"}` ✅
3. **BUT**: The live workflow does NOT write any correction file to OpenClaw workspace ❌
4. **AND**: The live workflow does NOT call OpenClaw `/observe` endpoint ❌

### Root cause

The **active n8n workflow is an empty stub** — it immediately returns `{"status":"success"}` without doing any actual work. The actual implementation in `services/n8n-workflows/conversation-correction-process.json` (write correction file → call OpenClaw `/workspace/{agentId}/write` → call `/observe`) has **not been deployed to n8n**.

### Verified pipeline works (for crypto plato user)

When I triggered a test with a valid UID (crypto plato's `5aGC5YE9BnhcSoTxxtT4ar6ILQy2`), the OpenClaw provision API logs show:
- `PUT /workspace/ella-omi-5agc5ye9bnhcsotxxtt4ar6ilqy2/write` → 200 OK
- `POST /workspace/ella-omi-5agc5ye9bnhcsotxxtt4ar6ilqy2/observe` → 200 OK
- Observer ran: prompt ~19K chars, model `minimax/MiniMax-M2.7`
- But Observer returned `[]` — no actions taken (likely because the correction file was never actually written)

### The fix needed

Deploy the `conversation-correction-process.json` workflow to n8n. This requires SSH access to `root@100.101.168.91` to update the n8n workflow.

### What works end-to-end today (for provisioned users)
- iOS → OMI backend `POST /v1/ella/conversations/{id}/corrections` ✅
- OMI backend → n8n `conversation-correction` webhook ✅
- n8n immediately returns 202 ✅
- OpenClaw Observer is reachable from n8n ✅
- BUT n8n workflow doesn't execute its logic (empty stub) ❌

**Pull request merged**